opencl/opencl-aot/source/main.cpp

```diff
@@ -322,37 +322,36 @@ int main(int Argc, char *Argv[]) {
                         cl::desc("Input OpenCL program binary files"));
   cl::alias OptIR("ir", cl::aliasopt(OptOutputElf), cl::desc("Output file"));
 
-  enum ArchType : int8_t { sse42 = 0, avx, avx2, avx512 };
+  enum ArchType : int8_t {
+    wsm,
```

Comment:
I'm pretty sure these new options will ruin customers applications. Customers use old values of `-march` option, and it's too tough to remove them in the single release.
For the current release, can you keep the old values, link them to the new architectures and print a warning highlighting that these old values will be removed in the next release and suggest how users should replace old values with the new ones? And in the next release you can remove these old options completely.

Reply:
Thanks! Restore old options. This PR will only add more options for march. Removal of old options maybe need further discussion and could be done on follow-up patch if the idea is approved.